### ISSUE TITLE
Do you know how to inject CSS before show WebView ??

### ISSUE THREAD

**Issue by QIWANG-2015** [haliccatalin/WebViewInjectingCSS#1]
Do you know how to inject CSS before show WebView ??

**Comment by QIWANG-2015** [comment]
It doesn't work .

**Comment by haliccatalin** [comment]
I tried to use method onPageStarted(), but after my page with CSS is loaded it is override with the initial page. In this case with Google page.

**Comment by QIWANG-2015** [comment]
Me too , but it is not enough . Thanks for your answer .

**Comment by QIWANG-2015** [comment]
I made it :) .

**Comment by haliccatalin** [comment]
Greate! Could you share it, please?

**Comment by QIWANG-2015** [comment]
Yes, of course . I will share it  later , I Got something urgent to do right now .

**Comment by QIWANG-2015** [comment]
I shared it on my Github . You can find it by this url :  **_https://github.com/QIWANG-2015/InjectCSSToWebView_** .

**Comment by haliccatalin** [comment]
Ok, thank you!
